### ISSUE TITLE
Text line width hard to scan

### ISSUE THREAD

**Issue by snacktavish** [OpenTreeOfLife/opentree#1212]
e.g. https://tree.opentreeoflife.org/opentree/argus/ottol@3581696

Could we make the font bigger or add line breaks? It's a bit hard to parse as single wide line.

**Comment by jimallman** [comment]
Yes, I can limit the overall width of this text to make it more legible. I'll add this change to the PR for showing conflict-tree details, already in progress.
